spec/summary.v1.yaml

```diff
@@ -215,6 +216,30 @@ components:
           type: number
           example: 5
           x-go-type: int
+        hasTimeInExtremeHighPercent:
+          type: boolean
+        timeInExtremeHighPercent:
+          description: Percentage of time spent in extreme high glucose range
```

Comment:
As I've mentioned in other places, I think it might be useful to list the range of values in the description at least (appears to be [0-1] here) just to help people out.

Not a blocker.

Reply:
...and there are several percentage documentation areas added, if you choose to use my suggestion.